test/project_types/theme/tasks/ensure_themekit_installed_test.rb

```diff
@@ -0,0 +1,73 @@
+# frozen_string_literal: true
+require 'project_types/theme/test_helper'
+
+module Theme
+  module Tasks
+    class EnsureThemekitInstalledTest < MiniTest::Test
+      def setup
+        super
+        @context = TestHelpers::FakeContext.new
+      end
+
+      def test_does_nothing_if_themekit_installed
+        File.expects(:exist?).with(EnsureThemekitInstalled::FILENAME).returns(true)
+        EnsureThemekitInstalled.call(@context)
```

Comment:
I may have the wrong end of the stick here, but will this test that we actually do nothing after the file check? Maybe we could check that `Net::HTTP.get` never gets called?

Reply:
Webmock will throw an error if a web request is made that has not been stubbed.

Reply:
We could perhaps use `assert_nothing_raised` to make this clearer.